docs/UserGuide.adoc

```diff
@@ -149,6 +149,8 @@ NOTE: Assume 2nd command occurs after 1st
 [[TagTag]]
 === Tagging a transaction: `tag`
 
+The `tag` command allows you to tag a `Transaction` entry.
```

Comment:
`**Transaction**`

Reply:
Not sure if it has been standardised in the document yet. We check again,